.travis.yml

```diff
@@ -1,8 +1,11 @@
 language: python
-services:
-- docker
 script:
 - make test
+install:
+  - docker exec -t asym-crypto-yaml bash -c "
```

Comment:
Since this is a private repo I don't think we have the image in dockerhub anymore, so you may need to build it from the dockerfile.  I also don't think it's a good idea to exec into a container like this if you can make a new image that will have what you need.